### DISCUSSION TITLE
Where can I read about self-correcting titles

### DISCUSSION THREAD

**Discussion by aviskase** [radekkozak/discordian#31] [Q&A] [answered]
Hi, changelog for v0.7.0 has

> Links in headers with preserved colors: this makes great sense when using Bryan Jenks (@tallguyjenks) idea of self-correcting titles

Bryan has tons of content =) Can you please point me to the source of this idea?

**Comment by tallguyjenks** [comment, marked as answer]
hey @aviskase I mentioned it here: https://github.com/BryanJenksCommunity/FAQ/discussions/19

**Comment by radekkozak** [comment]
Hi Yuliya, sorry for the delay, i was looking myself for the actual place where i saw it. I agree, Bryan has a lot of content, and very much helpful, so many thanks here @tallguyjenks for chiming in with quick response. I thought i heard about that in some of your videos actually, but yeah... hehe. Thanks ! 

/cc @aviskase

**Reply by tallguyjenks** [reply]
@radekkozak i do mention it in my videos, i just dont remember where 😅  but i do like to get responses in my [FAQ Repo](https://github.com/BryanJenksCommunity/FAQ/discussions) so I can just link them like I did here, makes for easy answering
